examples/gnrc_networking/Makefile

```diff
@@ -31,6 +31,14 @@ USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
 
+# added for compile-testing only
+# remove this as it bloats the default networking example
+USEMODULE += netstats_neighbor_etx
+USEMODULE += netstats_neighbor_count
+USEMODULE += netstats_neighbor_rssi
+USEMODULE += netstats_neighbor_lqi
+USEMODULE += netstats_neighbor_tx_time
```

Comment:
Please remove ;-).

Reply:
A test for this would be nice, when removed :-). For now it could literally be this application copied to `tests/netstats_neighbor`, but some unittest-style functions for the `netstats_neighbor` module would be desirable (and given its modular nature pretty straight forward).

Reply:
In #14623 I'm adding `examples/gnrc_networking_rpl`, I'll move this to `tests/` for now.